qctrlopencontrols/dynamic_decoupling_sequences/predefined.py

```diff
@@ -725,36 +848,71 @@ def new_quadratic_sequence(
     pre_post_rotation=False,
     **kwargs
 ):
-    """
-    Creates Quadratic Decoupling Sequence.
+    r"""
+    Creates the quadratic sequence.
 
     Parameters
     ----------
     duration : float, optional
-        defaults to None
-        The total duration of the sequence
-    number_outer_offsets : int, optional
-        Number of outer X-pi Pulses. Defaults to None. Not used if number_of_offsets
-        is supplied.
+        Defaults to None.
+        The total duration of the sequence :math:`\tau`.
     number_inner_offsets : int, optional
-        Number of inner Z-pi Pulses. Defaults to None. Not used if number_of_offsets
-        is supplied
+        Number of inner Z :math:`\pi`-pulses :math:`n_1`. Defaults to None.
+        Not used if number_of_offsets is supplied.
```

Comment:
I know this was just retained from before, but you could just remove it while you're here (since I don't think it's right). Same below.

Reply:
Removed now. Was planning to clean these while fixing the default values.